composer.json

```diff
@@ -11,7 +11,11 @@
         "drupal/linkit": "^6.0-beta1",
         "drupal/metatag": "^1.14",
         "drupal/pathauto": "^1.8",
+        "drupal/preview_link": "^1.7",
         "drupal/redirect": "^1.6",
+        "drupal/responsive_preview": "^1.0",
+        "drupal/role_delegation": "^1.1",
+        "drupal/scheduled_transitions": "^2.1",
```

Comment:
@stephen-cox are these the projects we want to remove from localgov_core in favour of https://github.com/localgovdrupal/localgov_workflows/blob/1.x/composer.json#L10 ?

Reply:
Should we also use [composer suggest ](https://getcomposer.org/doc/04-schema.md#suggest)here to reccomend users install localgov_workflows or will that be covered in the profile update.

Reply:
Good catch - PR for removing these here #114